ssg/build_remediations.py

```diff
@@ -298,6 +298,13 @@ def parse_from_file_with_jinja(self, env_yaml):
                 if platform in self.local_env_yaml["platform_package_overrides"]:
                     platform = self.local_env_yaml["platform_package_overrides"].get(platform)
 
+                    # Workaround for plaforms that are not Package CPEs
+                    # Skip platforms that are not about packages installed
+                    # These should be handled in the remediation itself
+                    print(platform)
```

Comment:
Probably forgotten `print`
It spams build output, see https://jenkins.complianceascode.io/job/scap-security-guide-pull-requests/4611/label=fedora/consoleFull
```
...
[114/371] [fedora-content] generating kubernetes-fixes.xml
Merged 70 kubernetes remediations.
[115/371] [fedora-content] collecting all bash fixes
Collected 472 bash remediations.
sssd-common
grub2-common
grub2-common
grub2-common
...
```

Reply:
Thanks, fixed.